recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -12,7 +12,7 @@ WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('PREFERRED_PROVIDER_virt
 
 RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'mfrlibs-rpi', 'mfrlibs-rpi', '', d)}"
 RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'broadcom-mfrlib', 'broadcom-mfrlib', '', d)}"
-RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'fwupgrade-lib', 'fwupgrade-lib', '', d)}"
+RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrfwlib', 'fwupgrade-lib', 'fwupgrade-lib', '', d)}"
```

Comment:
At line 9, shouldn't you change it also there? Can the PREFERRED_PROVIDER_virtual/mfrlib contain fwupgrade-lib?
Moreover, is there any possibilities that both PREFERRED_PROVIDER_virtual/mfrfwlib and PREFERRED_PROVIDER_virtual/mfrlib be available simultaneously? If yes then recipe will have additional not necessary dependency

Reply:
Update @artur-gebicz-ml